app/views/todo_list/edit.html.erb

```diff
@@ -2,7 +2,7 @@
 <h1>Edit</h1>
 
 <%= form_with model: @todo_list, local: true do |f| %>
-    <%= render(:partial => "form", :locals => {:f => f})%>
+    <%= render(:partial => 'form', :locals => {:f => f})%>
```

Comment:
this can be rewritten like this `<%= render 'form', f: f %>` and use more descriptive variables, avoid to use `f`